cmd/skupper/skupper_podman_site.go

```diff
@@ -50,13 +51,19 @@ func (s *SkupperPodmanSite) Create(cmd *cobra.Command, args []string) error {
 			Mode:     initFlags.routerMode,
 			Platform: types.PlatformPodman,
 		},
-		RouterOpts:                 routerCreateOpts.Router,
-		IngressHosts:               s.flags.IngressHosts,
-		IngressBindIPs:             s.flags.IngressBindIPs,
-		IngressBindInterRouterPort: s.flags.IngressBindInterRouterPort,
-		IngressBindEdgePort:        s.flags.IngressBindEdgePort,
-		ContainerNetwork:           s.flags.ContainerNetwork,
-		PodmanEndpoint:             s.flags.PodmanEndpoint,
+		IngressHosts:                 s.flags.IngressHosts,
+		IngressBindIPs:               s.flags.IngressBindIPs,
+		IngressBindInterRouterPort:   s.flags.IngressBindInterRouterPort,
+		IngressBindEdgePort:          s.flags.IngressBindEdgePort,
+		IngressBindFlowCollectorPort: s.flags.IngressBindFlowCollectorPort,
+		ContainerNetwork:             s.flags.ContainerNetwork,
+		PodmanEndpoint:               s.flags.PodmanEndpoint,
+		EnableFlowCollector:          routerCreateOpts.EnableFlowCollector,
+		EnableConsole:                routerCreateOpts.EnableConsole,
+		ConsoleUser:                  routerCreateOpts.User,
```

Comment:
Suggest adding console auth to allow for unsecured configuration if desired

Reply:
Done.